composer.json

```diff
@@ -17,14 +17,15 @@
 			"url": "git@github.com:WordPress/wporg-repo-tools.git"
 		}
 	],
-	"require": {
-		"composer/installers": "~1.0"
-	},
 	"require-dev": {
+		"composer/installers": "~1.0",
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
 		"phpcompatibility/phpcompatibility-wp": "*",
+		"phpunit/phpunit": "^9.5",
 		"wp-coding-standards/wpcs": "2.*",
-		"wporg/wporg-repo-tools": "dev-trunk"
+		"wp-phpunit/wp-phpunit": "~6.0",
```

Comment:
I'm pretty sure we don't need `phpunit/phpunit` or `wp-phpunit/wp-phpunit` since it's using the `wp-env` container & wp installation, but I've left it in from #204 in case it's necessary for running phpunit locally.